Ginger/Ginger/Reports/HTMLReportTemplatePage.xaml

```diff
@@ -111,6 +111,15 @@
                                         <RadioButton x:Name="htmlShowFirstIterationOnRadioBtn" Content="Yes" Padding="5,0,0,0" Checked="htmlShowFirstIterationOnRadioBtn_Checked" Style="{StaticResource @InputRadioButtonStyle}"/>
                                     </StackPanel>
                                 </StackPanel>
+                                <StackPanel x:Name="htmlExecutionCalculationLevel" Margin="10,10,0,0" Orientation="Horizontal" ToolTip="Show all iterations execution results on HTML Report e.g. Flow control or retry mechanism iterations">
+                                    <Label Content="Execution/Pass Rate Calculation Level:" Style="{StaticResource @LabelStyle}" FontWeight="Bold" FontSize="12"/>
+                                    <StackPanel x:Name="htmlExecutionCalculationLevelBtnsPnl" Margin="10,0,0,0" Grid.Row="0" Grid.Column="0" Orientation="Horizontal" HorizontalAlignment="Center" VerticalAlignment="Center">
+                                        <RadioButton x:Name="htmlExecutionCalculationLevelActivityRadioBtn" Tag="Activity" Content="Activity" Padding="5,0,15,0" Checked="htmlExecutionCalculationLevelActivityRadioBtn_Checked" Style="{StaticResource @InputRadioButtonStyle}"/>
```

Comment:
instead using tags just check the value of config field in init function and select the correct radio btn.
because tags must align with enum value of the field and we can't place it static here in xaml